src/dbnode/storage/index/types.go

```diff
@@ -74,44 +74,48 @@ func (o QueryOptions) LimitExceeded(size int) bool {
 	return o.Limit > 0 && size >= o.Limit
 }
 
-// QueryResults is the collection of results for a query.
-type QueryResults struct {
+// QueryReturnResults is the collection of results for a query.
+type QueryReturnResults struct {
```

Comment:
nit: Any huge reason to change QueryResults to QueryReturnResults?  Usually we try to keep language keywords like "return" from the names of types.